src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -3373,4 +3389,70 @@ private function matches() {
 	 * @access private
 	 */
 	const STATE_FUNKY_COMMENT = 'STATE_WP_FUNKY';
+
+	/**
+	 * Indicates that a comment was created when encountering abruptly-closed HTML comment.
+	 *
+	 * Example:
+	 *
+	 *     <!-->
+	 *     <!--->
+	 *     <!---->
+	 *
+	 * @since 6.5.0
+	 */
+	const COMMENT_AS_ABRUPTLY_CLOSED_COMMENT = 'COMMENT_AS_ABRUPTLY_CLOSED_COMMENT';
+
+	/**
+	 * Indicates that a comment would be parsed as a CDATA node,
+	 * were HTML to allow CDATA nodes outside of foreign elements.
```

Comment:
```suggestion
	 * were HTML to allow CDATA nodes outside of foreign content.
```

Reply:
accepted in 5e017d1bcfb077d42957ad76a631021cb8bc843a